spacelift.pkr.hcl

```diff
@@ -14,7 +14,27 @@ variable "ami_regions" {
 }
 
 variable "base_ami" {
-  type = string
+  type    = string
+  default = null
+}
+
+variable "source_ami_filters" {
+  type    = map(string)
+  default = {
+    virtualization-type = "hvm"
+    name                = "amzn2-ami-hvm-2*-x86_64-gp2"
+    root-device-type    = "ebs"
+  }
+}
+
+variable "source_ami_owners" {
+  type    = list(string)
+  default = ["137112412989"]
```

Comment:
`137112412989` looks suspiciously like a "magic number". Care to document?

Reply:
That owner ID is owned by Amazon itself. Unfortunately I cannot find a good reference for all amazon account numbers. I do see the account id referenced on this page.

https://docs.aws.amazon.com/AmazonECR/latest/userguide/amazon_linux_container_image.html

It could be documented as an inline comment in the hcl.

Reply:
Added an inline comment